vector/src/main/java/fr/gouv/tchap/sdk/session/room/model/RoomRetention.kt

```diff
@@ -0,0 +1,33 @@
+package fr.gouv.tchap.sdk.session.room.model
+
+import com.google.gson.annotations.SerializedName
+import fr.gouv.tchap.util.DinsicUtils
+import org.matrix.androidsdk.rest.model.Event
+import org.matrix.androidsdk.core.JsonUtils
+
+const val EVENT_TYPE_STATE_ROOM_RETENTION = "m.room.retention"
+
+const val STATE_EVENT_CONTENT_MAX_LIFETIME = "max_lifetime"
+const val STATE_EVENT_CONTENT_EXPIRE_ON_CLIENTS = "expire_on_clients"
+
+const val DEFAULT_RETENTION_VALUE = 365
```

Comment:
DEFAULT_RETENTION_VALUE_IN_DAYS, I like when the constant contains the unity

Reply:
ok